_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,133 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can send a variety of notifications to alert you
+of Forseti events. You can configure notifications to send to different
+channels and in different formats.
+
+### Notification Types
+
+  1. Inventory summary: Count of the resources added in the latest inventory crawl.
+  1. Violations: Individual violations that have been found from the latest scanner run. 
+  1. Cloud Security Command Center (Cloud SCC) findings: Violations converted to Cloud SCC findings format, that's ingestable by Cloud SCC.
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. Cloud Storage
+
+### Notification Formats
+
+  1. Human-readable data: CSV
+  1. Structured data: JSON
```

Comment:
In some cases in this document you describe the key on the same line, e.g.
```
`key`: Description
```
Other times you switch it:
```
Description: `value`
```
Other times you do both
```
`key`: `value`

Description.
```

```
`key`: `value`. Description.
```
Please just use one consistent format throughout.

Reply:
Done